tests/hap/template_svm_demo.py

```diff
@@ -155,17 +159,20 @@ def test_svm_wcs(gather_output_data):
     # Check the output primary WCSNAME includes FIT_SVM_GAIA as part of the string value
     tdp_files = [files for files in gather_output_data if files.lower().find("total") > -1 and files.lower().endswith(".fits")]
 
-    for tdp in tdp_files:
-        wcsname = fits.getval(tdp, "WCSNAME", ext=1).upper()
-        print("\ntest_svm_wcs.  WCSNAME: {} Output file: {}".format(wcsname, tdp))
-        assert WCS_SUB_NAME in wcsname, f"WCSNAME is not as expected for file {tdp}."
+    # This check is for all total data products which have the same "type" of WCSNAME -
```

Comment:
The test no longer verifies that the WCSNAME is coming out aligned to GAIA with this most recent change.  Having one test check for WCS_SUB_NAME insures that the alignment is coming out as expected, while checking to see that they are all the same can pass even if they are all -HSC30 apriori WCSs, which may actually indicate a failure to align.  This will work just fine, but the purpose of this test is now more limited and really most useful for validating the WCSs between detectors.

Reply:
I am OK with this as my intention is to update this "template" with the new tests we have put into the individual dataset tests so in the future one can just mine this file for ideas.  The upgrade should entirely be its own ticket.